src/Wrapper.js

```diff
@@ -8,12 +8,23 @@ const checkedProps = {
   onMenuToggle: PropTypes.func,
   onSelection: PropTypes.func.isRequired,
   closeOnSelection: PropTypes.bool,
-  tag: PropTypes.string
+  tag: PropTypes.string,
+  focusGroupOptions: PropTypes.object
 };
 
 class AriaMenuButtonWrapper extends React.Component {
   static propTypes = checkedProps;
-  static defaultProps = { tag: 'div' };
+  static defaultProps = {
+    tag: 'div',
+    focusGroupOptions: {
+      wrap: true,
+      stringSearch: true,
+      keybindings: {
+        next: { key: 'ArrowDown' },
+        prev: { key: 'ArrowUp' }
+      }
+    }
```

Comment:
Two outstanding problems I see here:
- We don't want to repeat these default props in two places. They could fall out of sync.
- If user wants to change *one* of these, they have to change *all* of them. A user should be able to, for example, set `stringSearch: true` without then having to set the keybindings, too. I think instead of using React's `defaultProps`, we should have a function that applies defaults more carefully to `focusGroupOptions`. This function could be used in multiple places, also, alleviating the first problem.